src/shared/globals/globals.js

```diff
@@ -24,6 +24,7 @@ import { translate, translatePlural } from '@nextcloud/l10n'
 
 import { appData } from '../../app/AppData.js'
 import { dialogs } from './OC/dialogs.js'
+import { MimeTypeList } from './OC/MimeType.js'
```

Comment:
incorrect filename, buiuld is failing

Reply:
I though `eslint-pluing-import` should have caught this 🙈